test/mako/alerter/alerter.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alerter
+
+import (
+	qpb "github.com/google/mako/proto/quickstore/quickstore_go_proto"
+	"knative.dev/pkg/test/mako/alerter/github"
+	"knative.dev/pkg/test/mako/alerter/slack"
+)
+
+// Alerter controls alert for performance regressions detected by Mako.
+type Alerter struct {
+	githubIssueHandler  *github.IssueHandler
+	slackMessageHandler *slack.MessageHandler
+}
+
+// SetupGithub will setup Github for the alerter.
```

Comment:
Why not just hide all this and the next one in a `NewAlerter`?

Reply:
If we combine them together, we will have a function like `NewAlerter(org, repo, githubTokenPath, slackUserName, slackReadTokenPath, slackWriteTokenPath string)`, which seems to have too many args to maintain..